website_docs/getting-started.md

```diff
@@ -0,0 +1,156 @@
+Collector.
+
+## Deployment
+
+The OpenTelemetry Collector consists of a single binary and two primary deployment methods:
+
+- **Agent:** A Collector instance running with the application or on the same
+  host as the application (e.g. binary, sidecar, or daemonset).
+- **Gateway:** One or more Collector instances running as a standalone service
+  (e.g. container or deployment) typically per cluster, datacenter or region.
+
+### Agent
+
+It is recommended to deploy the Agent on every host within an environment. In
+doing so, the Agent is capable of receiving telemetry data (push and pull
+based) as well as enhancing telemetry data with metadata such as custom tags or
+infrastructure information. In addition, the Agent can offload responsibilities
+that client instrumentation would otherwise need to handle including batching,
+retry, encryption, compression and more. OpenTelemetry instrumentation
+libraries by default export their data assuming a locally running Collector is
+available.
+
+### Gateway
+
+Additionally, a Gateway cluster can be deployed in every cluster, datacenter,
+or region. A Gateway cluster runs as a standalone service and can offer
+advanced capabilities over the Agent including tail-based sampling. In
+addition, a Gateway cluster can limit the number of egress points required to
+send data as well as consolidate API token management. Each Collector instance
+in a Gateway cluster operates independently so it is easy to scale the
+architecture based on performance needs with a simple load balancer. If a
+gateway cluster is deployed, it usually receives data from Agents deployed
+within an environment.
+
+## Getting Started
+
+### Demo
+
+Deploys a load generator, agent and gateway as well as Jaeger, Zipkin and
+Prometheus back-ends. More information can be found on the demo
+[README.md](https://github.com/open-telemetry/opentelemetry-collector/tree/main/examples/demo)
+
+```bash
+$ git clone git@github.com:open-telemetry/opentelemetry-collector.git; \
+    cd opentelemetry-collector/examples/demo; \
+    docker-compose up -d
+```
+
+### Docker
+
+Every release of the Collector is published to Docker Hub and comes with a
+default configuration file.
+
+```bash
+$ docker run otel/opentelemetry-collector
+```
+
+In addition, you can use the local example provided. This example starts a
+Docker container of the
+[core](https://github.com/open-telemetry/opentelemetry-collector) version of
+the Collector with all receivers enabled and exports all the data it receives
+locally to a file. Data is sent to the container and the container scrapes its
+own Prometheus metrics.
+
+```bash
+$ git clone git@github.com:open-telemetry/opentelemetry-collector.git; \
+    cd opentelemetry-collector/examples; \
+    go build main.go; ./main & pid1="$!";
+    docker run --rm -p 13133:13133 -p 14250:14250 -p 14268:14268 \
+      -p 55678-55680:55678-55680 -p 8888:8888 -p 9411:9411 \
```

Comment:
If I understand the spec correctly, it's only the gRPC port that needs to be updated. Please correct me if I'm wrong here.

```suggestion
      -p 55678-55679:55678-55679 -p 4317:4317 -p 8888:8888 -p 9411:9411 \
```